release/artifacts.yaml

```diff
@@ -33,15 +33,19 @@
   type: jar-scala
 - target: //daml-lf/data-scalacheck:data-scalacheck
   type: jar-scala
+- target: //daml-lf/encoder:encoder
+  type: jar-scala
```

Comment:
Why do we need the encoder? This was really only intended for internal testing for things that are hard to produce via the daml compiler.